scripts/test

```diff
@@ -70,7 +67,7 @@ compose() {
   fi
 
   # Run tests
-  compose run ${containerName}
+  compose run ffc-demo-user-service
```

Comment:
The service name was declared in a variable at the top for consistency with other FFC demo repos. It means we can easily copy changes between repos because the scripts in each are identical apart from the variable declarations at the top.

Reply:
I've reinstated the variable and appended '-test' for the project